framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -313,21 +318,23 @@ def index(self):
         portlets_list = []  # self.burst_service.get_available_portlets()
         template_specification['portletList'] = portlets_list
         template_specification['selectedPortlets'] = json.dumps(portlets_list)
+        is_branch = common.get_from_session(common.KEY_IS_SIMULATOR_BRANCH)
 
         form = self.prepare_first_fragment()
         rendering_rules = SimulatorFragmentRenderingRules(form, SimulatorWizzardURLs.SET_CONNECTIVITY_URL, None,
                                                           is_simulator_copy, is_simulator_load,
                                                           last_form_url=self.last_loaded_form_url,
                                                           last_request_type=cherrypy.request.method,
-                                                          is_first_fragment=True)
-
+                                                          is_first_fragment=True, is_branch=is_branch)
         template_specification.update(**rendering_rules.to_dict())
         return self.fill_default_attributes(template_specification)
 
-    def prepare_first_fragment(self):
+    def prepare_first_fragment(self, burst_config=None):
```

Comment:
burst_config is already in session at this point, right? Then, there is no need to pass it as an argument here, just take it from the session.